src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -165,8 +134,12 @@ public IReadOnlyList<SourceRepository> GlobalPackageFolderRepositories
 
                     // Read package folders from settings
                     var pathContext = NuGetPathContext.Create(Settings);
-                    var folders = new List<string>();
-                    folders.Add(pathContext.UserPackageFolder);
+                    var count = (pathContext.FallbackPackageFolders?.Count() ?? 0) + 1;
```

Comment:
why this has been changed?

Reply:
initializing a list with a fixed size can be efficient. You can read about it [here](https://msdn.microsoft.com/en-us/library/dw8e0z9z.aspx#Anchor_2).

Reply:
AFAIK it's efficient when it'd need to be increase the size and change the capacity. I don't think that's the case here.